src/api/api.ts

```diff
@@ -0,0 +1,47 @@
+import axios from 'axios';
+import { GoogleMessage } from '../models/GoogleMessage';
+import { MessageToRender } from '../models/MessageToRender';
+import { getUsefulMessageFields } from '../utils/getUsefulMessageFields';
+
+const getMessages = async (
+  authToken: string,
+  // todo
+  // filter: unknown = {},
+  // pageToken?: string,
+): Promise<Array<string>> => {
+  const { data } = await axios.get(
```

Comment:
You should use axios `param` property to send query params. see https://github.com/axios/axios